### PR TITLE
Adding Random Number Guesser

### PR DESCRIPTION
**Pull request by belikegt**

### CODE REVIEW

File: Giftson/randomNumGuesser.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
+"""
+Python Program to guess the number within the range 1-10
+Uses the library random
+Program Exp
+"""
+import random
+
+# using randint function from the random library to generate integer values
+# randint(1,10) produces a random value within the range of 1 to 10
+n = random.randint(1, 10)
```
Comment (Rangeeshar) [inline Giftson/randomNumGuesser.py]: variable naming should be human readable terms.

File: Giftson/randomNumGuesser.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+# randint(1,10) produces a random value within the range of 1 to 10
+n = random.randint(1, 10)
+
+print("Please Guess the Number!")
+
+# loop will continue to run until the break statement runs.
+while True:
+    o = int(input())
+    if n < o:
+        print("The number you entered is larger\n Please guess the number again")
```
Comment (Rangeeshar) [inline Giftson/randomNumGuesser.py]: add like hint: this comment same for other hint also.

**Review by Rangeeshar — CHANGES_REQUESTED** [review]
changes needed

File: Giftson/randomNumGuesser.py [hunk 3]
```diff
@@ -0,0 +1,23 @@
+"""
+Python Program to guess the number within the range 1-10
```
Comment (Rangeeshar) [inline Giftson/randomNumGuesser.py:2]: Add comments how to run this code

**Review by Rangeeshar — CHANGES_REQUESTED** [review]
changes

File: Giftson/randomNumGuesser.py (hunk 2 above)
Comment (belikegt) [reply Giftson/randomNumGuesser.py]: anna im not getting what you are trying to say here anna

Comment (Rangeeshar) [reply Giftson/randomNumGuesser.py]: print("hint: <your statement>")

**Review by Rangeeshar — CHANGES_REQUESTED** [review]
changes

File: Giftson/randomNumGuesser.py (hunk 2 above)
Comment (belikegt) [reply Giftson/randomNumGuesser.py]: > print("hint: ")

ow okay i get it now

**Review by Rangeeshar — APPROVED** [review]

**Comment by belikegt** [comment]
Changes made

**Pull request merged**